include/fbgemm/PackingTraits-inl.h

```diff
@@ -19,17 +54,21 @@ struct PackingTraits<
     std::int32_t,
     inst_set_t::avx2,
     typename std::enable_if<is_8bit<T>::value>::type> {
-  static constexpr int MR{12}; ///< Register block for M dimension
-  static constexpr int NR{8}; ///< Register block for N dimension
+  static constexpr int MR{12}; ///< Register block for M dimension.
+  static constexpr int NR{8}; ///< Register block for N dimension.
+                              ///< NR = VLEN/ROW_INTERLEAVE = 256 / 8 / 4 = 8.
+                              ///< Total registers used for N dimension: NCB/NR.
+                              ///< Here we use 12 x 1 ymm register blocking for
+                              ///< the registers used for accumulation C.
 
   static constexpr int ROW_INTERLEAVE{
       4}; ///< 4 rows are interleaved to use vpmaddubsw instruction for packing
-          ///< B matrix.
+          ///< B matrix. 4 * 8 (size of int8) = 32-bit accumulation.
```

Comment:
>  4 * 8 (size of int8) = 32-bit accumulation. 

Remove this. This isn't the reason. We combine 4 of them because underlying x86 instruction works nicely when combining 4 of them not because we are accumulating into 32-bits.